src/app/globals.css

```
@@ -2519,3 +2519,349 @@ select option {
+  --avatar-border: rgba(244, 236, 216, 0.3);
+  --avatar-color: #f4ecd8;
+  --gradient-title-start: #f4ecd8;
+  --gradient-title-end: #b5ab90;
+  --modal-backdrop: rgba(10, 8, 4, 0.8);
+}
+
+/* Newsprint Sudoku swaps digits + headings to a serif face so the theme
+   reads as newsprint print, not just cream-coloured. Confined to the
+   game-theme subtree so nothing else on the site catches serif. */
+[data-game-theme='sudoku-newsprint'] {
+  font-family: 'Georgia', 'Times New Roman', serif;
```

Stylelint (17.14.0)

[error] Expected no quotes around "Georgia" (font-family-name-quotes)

(font-family-name-quotes)